### PR TITLE
New fields in models

### PR DESCRIPTION
**Pull request by pegasusflyhigh**

### CODE REVIEW

File: app/models/student.rb [hunk 1]
```diff
@@ -27,9 +27,10 @@ class Student < ApplicationRecord
   belongs_to :school, class_name: "School", foreign_key: "school_id"
 
   validates :registration_no, :roll_no, :gender, :dob, :guardian_name,
-            :guardian_mobile_no, presence: true
+            :guardian_mobile_no, :message_language, presence: true
```
Comment (anilmaurya) [inline app/models/student.rb]: Instead of `message_language`, lets use `preferred_language` field_name.

Comment (pegasusflyhigh) [reply app/models/student.rb]: Okay

File: app/models/student.rb [hunk 2]
```diff
@@ -27,9 +27,10 @@ class Student < ApplicationRecord
...
 
   validates :registration_no, uniqueness: true
+  validates :message_language, inclusion: {in: %w[hindi english marathi]}
```
Comment (anilmaurya) [inline app/models/student.rb]: Lets save locale `en mr-IN hi-IN` instead of full text

Comment (pegasusflyhigh) [reply app/models/student.rb]: Yes

**Pull request merged**